monitoring.yaml

```diff
@@ -0,0 +1,12 @@
+user_name: "USERNAME"
+password: "AUTH"
+components:
+  - repo_name: "tidb"
+    monitor_path: "metrics/grafana"
+    rule_path: "metrics/alertmanager"
+  - repo_name: "tikv"
```

Comment:
tikv repo is in tikv organization. Does this work?

Reply:
Currently, No components merge the PR to master.
So i have to test it in ansible, it works well, the configuration like this:
```
components:
  - repo_name: "tidb-ansible"
    monitor_path: "scripts"
    rule_path: "roles/prometheus/files"
```
It works well.

Reply:
Ah, okay I see you've added the `owner` field for tikv.